apps/builder/app/builder/features/style-panel/controls/color/color-control.tsx

```diff
@@ -45,25 +47,26 @@ export const ColorControl = ({
   deleteProperty,
   isAdvanced,
 }: ControlProps) => {
+  const $computedStyleDecl = useMemo(
+    () => createComputedStyleDeclStore(property),
+    [property]
+  );
+  const computedStyleDecl = useStore($computedStyleDecl);
```

Comment:
seems very unusual and mentally complex to generate a store in a hook and then memoize it in the component as well

is there anything we can do to simplify? If not, I think this is the case where it would be better in a custom hook, so we don't have to write it in every control.